### ISSUE TITLE
Retrieving datas from SUMO simulation started with Flow using TraCI from another terminal

### ISSUE THREAD

**Issue by LorenzoUccello** [eclipse/sumo#11749]
I am using flow to integrate deep reinforcement learning using a SUMO traffic simulation. 
My goal is to get two different terminals to access the same SUMO simulation: the first generating the SUMO simulation using Flow and the second using TraCI to grant access to the simulation and asking for vehicle data at each time step, such as position or speed. Unfortunately, something seems not to be working. My second terminal links to the simulation, but it exits as soon as the simulation starts without saving anything. TraCI doesn't see any vehicle and gives traci.simulation.getMinExpectedNumber() = 0, even if there are more than 10 vehicles.

Here below, I send the lines of code I use to initialise this communication:

**1. THIS IS THE CODE I USE ON THE FIRST TERMINAL TO ASSIGN THE PORT AND THE CLIENTS FOR SUMO WITH FLOW**

sim_params = SumoParams(port=45678, render=VISUALIZE_SIM_IN_SUMO, sim_step=1, num_clients=2)

**2. THIS IS THE CODE I USE ON THE SECOND TERMINAL TO CONNECT WITH TRACI AND RETRIEVE DATA**

traci.init(PORT,tc.DEFAULT_NUM_RETRIES,"second_computer.local")
traci.setOrder(2)

**SUMO Version: v1_14_1+0776-419d5e4252e**

**operating system: macOS Monterey version 12.6**

**Comment by namdre** [comment]
are you also calling setOrder in the first client?

**Comment by LorenzoUccello** [comment]
Hello Jakob, 

Thank you for your answer. I've just solved the problem. 
I cannot use setOrder on the first terminal, because Flow does not have any parameter to set it once I start the simulation. I can only set the port (but this command does not work properly) and the number of clients. 

Looking at the second terminal results, I saw that traci.simulation.getMinExpectedNumber() returns 0, since TraCi cannot see any vehicle in the simulation. I think that this depends on the fact that Flow does not load the sumocfg file, where the file with all the vehicles info is linked, but only the net file. It uses one of its methods to create vehicles in the simulation. 

This way the TraCI cycle can start on the second terminal and once the vehicles are in the simulation TraCI sees them and gives me all the data. 

Thank you again and have a nice evening!
